### PR TITLE
Build

### PR DESCRIPTION
**Pull request by vikaschoudhary16**
Revendoring and fixes to complete first build

```
CGO_ENABLED=0 GOOS=linux go build -o bin/machine-api-operator -a -ldflags '-extldflags "-static"' github.com/openshift/machine-api-operator/cmd
```

### CODE REVIEW

**Comment by ingvagabund** [comment]
> Revendoring and fixes to complete first build

What does it mean here to build? Just run `make build`?

File: Gopkg.toml [hunk 1]
```diff
@@ -25,21 +25,17 @@
...
-  branch = "master"
-
 [[constraint]]
   name = "github.com/ghodss/yaml"
   version = "1.0.0"
 
-[[constraint]]
-  name = "k8s.io/apimachinery"
-  version = "kubernetes-1.9.0"
+#[[constraint]]
```
Comment (enxebre) [inline Gopkg.toml]: remove this?

**Comment by enxebre** [comment]
lgtm other than the commented library if looks good to @ingvagabund

**Pull request merged**